README.md

```diff
@@ -15,20 +15,27 @@ For more information, please refers to [BUILDING.md](BUILDING.md).
 If you'd like to contribute, please review the
 [Contributing Guidelines](CONTRIBUTING.md).
 
-## End-to-End Testing
+## Testing
 ### Requirements
 
 - [Python3.6+](https://www.python.org/)
 - [tox](https://pypi.org/project/tox)
 - [Vagrant](https://www.vagrantup.com/)
 - [VirtualBox](https://www.virtualbox.org)
 
-### Usage
+### Bootstrapping local environment
 
-To run tests locally:
-```shell
+```shell 
+# Install virtualbox guest addition plugin
+vagrant plugin install vagrant-vbguest
 # Bootstrap a platform on a vagrant environment using
-./doit.sh vagrantup
+./doit.sh vagrant_up
+```
+
+### End-to-End Testing
+
+To run tests locally: Complete the bootstrapping step above
+```shell 
```

Comment:
While you're at it, we don't need to generate the SSH config file by hand anymore, it's handled in `tox`. You could update this code block with something like:

```shell
# Run tests with tox
tox -e tests
```

And optionally (but I think we could do it in another PR, when we extract the Terraform tooling), mention how one can define his own `SSH_CONFIG_FILE` and `SSH_HOSTS_LIST` environment vars to affect the `tox -e tests` invocation.

Reply:
Thanks for the heads-up @gdemonet